src/video/win32_v.cpp

```diff
@@ -397,6 +400,59 @@ static void CancelIMEComposition(HWND hwnd)
 	HandleTextInput(nullptr, true);
 }
 
+static bool IsDarkModeEnabled()
+{
+	/* Only build if SDK is Windows 10+. */
+#if defined(_MSC_VER) && defined(NTDDI_WIN10)
+	if (IsWindows10OrGreater()) {
```

Comment:
Could be an early `return false` instead of a nested if.

Reply:
I'd have to do #if/#else/#endif then though, or you'd have some unreachable code in some situations.